index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>my mobile porifolio</title>
+</head>
+<body>
+    <div class="container">
+        <div class="menu">
+            <div class="logo"><h2><span>My Logo</span></h2></div>
+            <div class="menu-bar"><i class="fa fa-bars fa-2x"></i>
```

Comment:
- [ ] Please wrap this menu with a `nav` element since it should be inside the nav as it is the menu for the nav links.